src/base.config.ts

```diff
@@ -336,7 +365,7 @@ export default function webpackConfigFactory(args: any): WebpackConfiguration {
 					include: allPaths,
 					test: /\.css$/,
 					exclude: /\.m\.css$/,
-					use: cssLoader
+					oneOf: [{ issuer: indexHtmlPattern, use: 'identity-loader' }, { use: cssLoader }]
```

Comment:
although this works and is neat (conditionally applying a loader on issuer), we need to fix these loader paths unrelated to the PR anyway as we shouldn’t be processing modules outside of `src` or `test` regardless of issuer. We currently have side effects elsewhere due to this (ie we are processing external/node_modules css through these loaders).